### DISCUSSION TITLE
Welcome to StableCanvas Discussions!

### DISCUSSION THREAD

**Discussion by zhzLuke96** [StableCanvas/StableCanvas#3] [Announcements]
## 👋 Welcome everyone!

We're launching the StableCanvas discussion section here, hoping it will facilitate conversation and exchange among all of you.

Here you can:
  * Discuss your issues and the bugs encountered.
  * Discuss your intended issues before actually submitting them.
  * Share your ideas and suggestions.
  * Share your creative projects.

About StableCanvas:
- StableCanvas can be considered as another GUI program for the stable diffusion series of models. Unlike traditional a111/webui or comfyui, this program focuses more on editing images and performing operations on the canvas - there's no backend - all of our program code runs in your browser.
- In short, you can consider this program as an alternative to Photoshop (in terms of AI), simplifying operations, making the AI creation process more reasonable and smooth, without the need to switch between different programs.

Jump in and join the conversation. We're excited for you to be a part of our growing community!
